group_vars/all/ignored_nics.yaml

```diff
@@ -0,0 +1,5 @@
+ignored_nics:
+  - lo
+  - docker0
+  - virbr0
+  - virbr1
```

Comment:
I wonder if it would make more sense to check if the address is contained by a known MOC CIDR range? This way to don't need to account for the the various bridge naming conventions used by docker, podman, libvirt, etc. E.g., while the default Docker bridge is `docker0`, docker will create a new bridge device `br-<something>` for every user-defined Docker network.

Reply:
That makes sense. Let me see if I can figure out how to do a check against the prefixes file https://github.com/CCI-MOC/moc-ansible-netbox/blob/main/group_vars/all/netbox_prefixes.yaml